### PR TITLE
testing with npm

### PR DESCRIPTION
**Pull request by mansona**
This is essentially an extension of https://github.com/ember-cli/ember-cli/pull/7810

I think it's potentially worth considering having at least one test run with npm only (no yarn) as a kind of smoke test for any future issues that might crop up 👍

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -59,6 +59,11 @@ jobs:
     - env: EMBER_CLI_BROCCOLI_2=false
     - env: EMBER_CLI_SYSTEM_TEMP=false
 
+    - env: NAME=NODE_10_NPM
```
Comment (Turbo87) [inline .travis.yml]: we can use `name: npm` now instead of misusing `env`

File: .travis.yml [hunk 2]
```diff
@@ -59,6 +59,11 @@ jobs:
...
+      node_js: 10
+      install: npm install
+      script: npm run test:all
```
Comment (Turbo87) [inline .travis.yml]: this sounds like it would add significantly more time to our already long CI pipeline 😢

File: .travis.yml (hunk 1 above)
Comment (mansona) [reply .travis.yml]: 👍

File: .travis.yml (hunk 2 above)
Comment (mansona) [reply .travis.yml]: I guess it depends on what you mean by "significant" 😂 

In the case of individual jobs it seems like the npm run is consistently faster than yarn in the 2 out of 2 builds that I've seen now 😂 Not a great sample size but considering that's without a package-lock.json I'm pretty impressed.

Secondly, it seems like we are getting a 5 job concurrency on the big stage and adding this job brings the total number of jobs to 10. In reality I guess that means that as long as we keep the job count below 10 we should be fine when it comes to elapsed time 🤔

**Comment by rwjblue** [comment]
Going to close this now, we no longer pin down npm version. Happy to review/help with another PR to master with GH actions if you still think we should move forward.

**Pull request closed without merging**